modules/olm-enabling-operator-restricted-network.adoc

```diff
@@ -88,3 +88,15 @@ spec:
 <1> Inject the images referenced by the Operator via environment variables.
 <2> Specify each image by a digest (SHA), not by an image tag.
 <3> Also reference the Operator container image by a digest (SHA), not by an image tag.
+
+* Look for the `Disconnected` annotation, which indicates that the Operator works
```

Comment:
```
Look for the `Disconnected` annotation
```

Am I misunderstanding? I think it should be `Look for the `operators.openshift.io/infrastructure-features` annotation`

Reply:
That annotation will be used for a number of different features. This doc is about disconnected, so the only relevant value is `disconnected`.

Reply:
Thanks for the explanation, sounds reasonable